app/helpers/application_helper.rb

```diff
@@ -8,4 +8,26 @@ def page_title(page)
       t('page_titles.application')
     end
   end
+
+  def service_name
+    case current_namespace
+    when 'provider_interface'
```

Comment:
This all seems a bit overengineerd, but we'll soon use it for the support section too (https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/272).